modules/efk-logging-deploy-subscription.adoc

```diff
@@ -35,13 +35,18 @@ You *must* install the Elasticsearch Operator using the CLI following the direct
 You can install the Cluster Logging Operator using the web console or CLI.
 ====
 
-
 .Procedure
 
-. Install the Elasticsearch Operator using the CLI to ensure the necessary values are set:
+. Create Namespaces for the Elasticsearch Operator and Cluster Logging Operator.
++
+[NOTE]
+====
+You can create the Namespaces in the web console using the *Administration* -> *Namespaces* page.
+====
 
-.. Create a Namespace for the Elasticsearch operator (for example, `eo-project.yaml`):
+.. Create a Namespace for the Elasticsearch Operator (for example, `eo-project.yaml`):
```

Comment:
would probably change the example to `eo-namespace.yaml` just to comply with us always using "namespace" and not "project".